composer.json

```diff
@@ -60,7 +60,7 @@
     "davidstutz/bootstrap-multiselect": "v0.9.13",
     "easyrdf/easyrdf": "0.10.0-alpha.1",
     "etdsolutions/waypoints": "4.0.0",
-    "twig/twig": "2.4.*",
+    "twig/twig": "2.12.*",
```

Comment:
> 
> 
> I'm struggling a bit with this PR...
> 
> First the apparent problem: there are two merge conflicts against `master` (see detailed comments) and they should be fixed before this can even be properly tested.

These are artifacts that result from this PR being open for so long.

> Second, I have the feeling again that this PR tries to do too many things at once. It's not crystal clear why it has to be like that. It would be better to fix problems individually in very small increments. That makes it easier to review, and also to backtrack if new problems are detected later.

I agree to some point. However, if one looks at the separate commits, one can see small increments being deployed (especially see their textual information that explains things). Additionally, this will result in exactly the same (if not squashing the PR!) situation later, if something is found to be off.

On a side note: if I recall correctly, I just happened to stumble upon all this **small** things when fixing this one thing. Previously, we would have just committed them directly but that is not the case anymore. And to be precise, you are correct, anyways, I give you that.

> Third, the PR makes it apparent that we are constructing very complex URLs using Twig expressions. This seems wrong-headed, and we should try to do that in PHP code instead, using the `link_url` filter if possible and if not, perhaps extending it and/or adding new similar filters. This isn't directly the fault of this PR though, the problem has existed before and this just makes it more visible.

Yes, that would make sense. I am a bit hesitant if this PR is the one that changes the course of the boat - we should have a more throughout conversation.